metals/src/main/scala/scala/meta/internal/metals/doctor/Doctor.scala

```diff
@@ -174,6 +177,9 @@ final class Doctor(
           hasProblems.set(true)
           languageClient.showMessageRequest(params).asScala.foreach { item =>
             if (item == CheckDoctor.moreInformation) {
+              if (clientConfig.isDoctorVisibilityProvider()) {
+                isVisible.set(true)
+              }
```

Comment:
When there is some problem there is also `More Information` button, without this snippet, it won't work.

Reply:
I broke this in https://github.com/scalameta/metals/pull/3768